notification/src/notificationEndpoint.ts

```diff
@@ -3,11 +3,15 @@ import express, { Application, Request, Response } from 'express'
 import session, { MemoryStore } from 'express-session'
 import cors from 'cors'
 import Keycloak from 'keycloak-connect'
-import NotificationService from './interfaces/notificationService'
- 
+import NotificationService from './interfaces/notificationService';
+import "reflect-metadata"
```

Comment:
Can be removed because imported in index?

Reply:
need it for type definitions in constructor parameter and member variable